src/index.ts

```diff
@@ -14,3 +14,6 @@ export { compile } from './core/compile';
 export { validateGoslingSpec } from './core/utils/validate';
 export { GoslingComponent } from './core/gosling-component';
 export { embed } from './core/gosling-embed';
+
+// Utils
+export { traverseTracksAndViews, traverseViewArrangements } from './core/utils/spec-preprocess';
```

Comment:
We could expose more util functions here later.

Reply:
Good to know! I wonder if we need to expose these currently to users.. We could treat these as private functions by importing from src:

```typescript
// editor.tsx
import { traverseTracksAndView } from 'gosling.js/core/utils/spec-preprocess';
```